dataactbroker/fileRoutes.py

```diff
@@ -102,3 +102,17 @@ def get_protected_files():
         """ Return signed URLs for all help page files """
         fileManager = FileHandler(request, isLocal=IS_LOCAL, serverPath=SERVER_PATH)
         return RouteUtils.run_instance_function(fileManager, fileManager.getProtectedFiles)
+
+    @app.route("/v1/generate_file/", methods=["POST"])
+    @permissions_check
+    def generate_file():
+        """ Generate file from external API """
+        fileManager = FileHandler(request, isLocal=IS_LOCAL, serverPath=SERVER_PATH)
+        return RouteUtils.run_instance_function(fileManager, fileManager.generateFile)
+
+    @app.route("/v1/check_generation_status/", methods=["POST"])
+    @permissions_check
```

Comment:
These routes were defined in https://gist.github.com/kevinli-work/59f7310afc789fdeeb7b0993772df03d, so I followed what was in that.  This route does take a couple parameters, which is why it's a post route.  We could make this a GET route and have the parameters be part of the URL, but that isn't really consistent with how we've done similar routes like /v1/check_status/.  If we want to change this pattern, it should probably be its own story to go through and identify the routes that should be done this way, with hours included for the necessary frontend changes to use the altered routes.